notebooks/Binary Classification with the UCI Credit-card Default Dataset.ipynb

```diff
@@ -1,710 +0,0 @@
-   "outputs": [],
-   "source": [
-    "# Plot equalized odds difference vs balanced accuracy\n",
-    "plt.scatter(balanced_accuracy_non_dominated, equalized_odds_sweep_non_dominated, label=\"GridSearch Models\")\n",
-    "plt.scatter(balanced_accuracy_score(Y_test, test_preds),\n",
-    "            equalized_odds_difference(Y_test, test_preds, sensitive_features=A_str_test), \n",
-    "            label=\"Unmitigated Model\")\n",
-    "plt.scatter(balanced_accuracy_score(Y_test, postprocess_preds), \n",
-    "            equalized_odds_difference(Y_test, postprocess_preds, sensitive_features=A_str_test),\n",
-    "            label=\"ThresholdOptimizer Model\")\n",
-    "plt.xlabel(\"Balanced Accuracy\")\n",
-    "plt.ylabel(\"Equalized Odds Difference\")\n",
-    "plt.legend(bbox_to_anchor=(1.55, 1))\n",
-    "plt.show()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "As intended, `GridSearch` models appear along the trade-off curve between the large balanced accuracy (but also large disparity), and low disparity (but worse balanced accuracy). This gives the data scientist a flexibility to select a model that fits the application context best."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Plot equalized odds difference vs AUC\n",
-    "plt.scatter(auc_non_dominated, equalized_odds_sweep_non_dominated, label=\"GridSearch Models\")\n",
-    "plt.scatter(roc_auc_score(Y_test, test_scores),\n",
-    "            equalized_odds_difference(Y_test, test_preds, sensitive_features=A_str_test), \n",
-    "            label=\"Unmitigated Model\")\n",
-    "plt.scatter(roc_auc_score(Y_test, postprocess_preds), \n",
-    "            equalized_odds_difference(Y_test, postprocess_preds, sensitive_features=A_str_test),\n",
-    "            label=\"ThresholdOptimizer Model\")\n",
-    "plt.xlabel(\"AUC\")\n",
-    "plt.ylabel(\"Equalized Odds Difference\")\n",
-    "plt.legend(bbox_to_anchor=(1.55, 1))\n",
-    "plt.show()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Similarly, `GridSearch` models appear along the trade-off curve between AUC and equalized odds difference."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Compare GridSearch models with low values of equalized odds difference with the previously constructed models\n",
-    "grid_search_dict = {\"GridSearch_{}\".format(i): (sweep_preds[i], sweep_scores[i])\n",
-    "                    for i in range(len(sweep_preds))\n",
-    "                    if non_dominated[i] and equalized_odds_sweep[i]<0.1}\n",
-    "models_dict.update(grid_search_dict)\n",
-    "get_metrics_df(models_dict, Y_test, A_str_test)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "source": [
-    "# Simplifying this analysis using model comparison plots\n",
```

Comment:
This seems to have been added somewhat recently via https://github.com/fairlearn/fairlearn/pull/947/files#diff-c9fd48863eb70895ec467c279a87b578a78a7318597e651561e57a52ef7d062e
Do we want to preserve this in some way?